site/en/r2/tutorials/load_data/numpy.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Please no functions within functions.</p><p><br></p><p>Also, </p><p><br></p><p>`return tf.reshape(x, [784]) / 255.`</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
> no functions in functions
okay

> `return tf.reshape(x, [784]) / 255.`

this causes a type error:

```
TypeError: x and y must have the same dtype, got tf.uint8 != tf.float32
```

Reply:
Removed altogether.